src/pages/docs/projects/project-triggers/git-triggers.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-05
+modDate: 2024-11-05
+title: Git repository triggers
+description: Git repository triggers allow you to automatically create a new release when a new commit is pushed to a Git repository.
+navOrder: 16
+icon: fa-brands fa-git-alt
+---
+
+## Getting started
+
+Navigate to your project and click **Triggers**. Click **Add Trigger** on the right-hand side of the page, and select **Git repository**.
+
+Enter a name and description for your trigger.
+
+## Channels and lifecycles
+
+:::div{.hint}
+Git repository triggers create releases based on the default branch in version controlled projects
+:::
+
+If your project contains multiple [channels](/docs/releases/channels), you have the option of selecting which channel this trigger will apply to. The releases created by the trigger will use this channel.
+
+The versions used for those releases is guided by [release versioning](/docs/releases/release-versioning) under **Settings**. They will use the rules defined there.
+
+A preview of the [lifecycle](/docs/releases/lifecycles) used by the selected channel is displayed. You can modify the [lifecycle's phases](/docs/releases/lifecycles/#Lifecycles-LifecyclePhases) to have a release created and deployed to selected environments whenever a new commit is pushed to the monitored repository.
```

Comment:
> You can modify the [lifecycle's phases](/docs/releases/lifecycles/#Lifecycles-LifecyclePhases) to have a release created and deployed to selected environments whenever a new commit is pushed to the monitored repository.

The way this is phrased makes it sounds kind of like you can modify the lifecycle in place, perhaps we could add something about clicking on the relevant link to modify the lifecycle if required?

Reply:
Added